bci_essentials/bci_data.py

```diff
@@ -667,8 +843,28 @@ def preprocessing(self, window, option=None, order=5, fc=60, fl=10, fh=50):
 
     # Artefact rejection goes here (windows are nchannels by nsamples)
     def artefact_rejection(self, window, option=None):
-        """
-        Howdy
+        """Artefact rejection.
```

Comment:
Minor, optional is missing the **optional** label to it

Reply:
Fixed